### ISSUE TITLE
ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-annotations:26.1.0.

### ISSUE THREAD

**Issue by varunon9** [tolu360/react-native-google-places#204]
I am getting following errors after upgrading gradle in android-

```
ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-compat:26.1.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-annotations:26.1.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve android.arch.lifecycle:runtime:1.0.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve android.arch.lifecycle:common:1.0.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve android.arch.core:common:1.0.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-core-utils:26.1.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-core-ui:26.1.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-fragment:26.1.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-annotations:28.0.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-compat:28.0.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-core-utils:28.0.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-fragment:28.0.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-compat:26.1.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-core-utils:26.1.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-core-ui:26.1.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-fragment:26.1.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve android.arch.lifecycle:runtime:1.1.1.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-core-ui:28.0.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve com.android.support:support-annotations:26.1.0.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve android.arch.lifecycle:common:1.1.1.
Show Details
Affected Modules: react-native-google-places

ERROR: Unable to resolve dependency for ':react-native-google-places@debug/compileClasspath': Could not resolve android.arch.core:common:1.1.1.
Show Details
Affected Modules: react-native-google-places
```

build.gradle (project)-

```
buildscript {
    ext {
        buildToolsVersion = "28.0.3"
        minSdkVersion = 16
        compileSdkVersion = 28
        targetSdkVersion = 27
        supportLibVersion = "28.0.0"
        googlePlayServicesVersion = "16.0.0"
    }
    repositories {
        google()
        jcenter()
        maven {
            url 'https://maven.fabric.io/public'
        }
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.3.2'
        classpath 'com.google.gms:google-services:4.0.1'
        classpath 'io.fabric.tools:gradle:1.26.1'


        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}
```

Adding this line manually in build.gradle of react-native-google-places solve the problem-

```
implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
```

**Comment by marcosandri-dev** [comment]
This pr is going to be accepted I think. If you use React Native > 59 try to copy the code into your node_modules build.gradle file, or fork this repo with the code and connect your package.json to it. 
https://github.com/tolu360/react-native-google-places/pull/202

**Comment by eng-abbas** [comment]
i do not understand

**Comment by wkwyatt** [comment]
@marcosandri-dev That node module update didn't work for me

**Comment by congnguyen91** [comment]
@wkwyatt  try to use my fork util my PR merged :) You should upgrade to API, buildTool 28

**Comment by tolu360** [comment]
PR is merged, please pull in `v3.0.1` and rebuild your project.
